src/shaders/MaterialX/color2.h

```diff
@@ -15,340 +15,287 @@ struct color2
 
 color2 make_color2(float r, float g)
```

Comment:
Is make_color2 even needed now that we have constructors? Is there any use of make_color(r,g) that should not simply be color2(r,g)?